app/mailers/feedback_submission_mailer.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+class FeedbackSubmissionMailer < ApplicationMailer
+  RECIPIENTS = %w(
+    feedback@va.gov
+  ).freeze
+
+  def build(feedback, github_link)
+    @feedback = feedback
+    @github_link = github_link
+    template = File.read('app/mailers/views/feedback_report.erb')
+
+    mail(
+      to: recipients,
+      subject: subject_line,
+      body: ERB.new(template).result(binding)
+    )
+  end
+
+  private
+
+  def recipients
+    FeatureFlipper.staging_email? ? 'bill.ryan@adhocteam.us' : RECIPIENTS.clone
+  end
+
+  def subject_line
+    subject = 'Vets.gov Feedback Received'
+    subject += ' - Response Requested' unless @feedback.owner_email.nil?
```

Comment:
minor :) but `<<` is an alias for `my_string.concat('another string')` which is faster than `+=` 
```ruby
subject << ' - Response Requested' unless @feedback.owner_email.nil?
```

Reply:
Getting a 
```
RuntimeError:
       can't modify frozen String
```

when using the `<<` operator probably due to the comment at the top:
```
# frozen_string_literal: true
```

It's a subtle difference; `<<` operates on the same instance, while `+=` returns a new one.

Reply:
https://stackoverflow.com/a/4684507